### PR TITLE
LRDOCS-9489 updates Editing Images article with 7.4 content

### PR DESCRIPTION
**Pull request by JamesAGarcia**
The following PR updates the existing Editing Images article with 7.4 content, per [LRDOCS-9489](https://issues.liferay.com/browse/LRDOCS-9489).

**General Question about Image Editor 7.4**:
I see that in 7.3.x, the Image Editor includes filters, and other editing tools that are removed in 7.4.
Q: Will these features be added again in the future, or is there no plan to incorporate them again?

**Questions about Zoom for SME**:
I was wondering if you could further explain how the zoom tool works.
The text states: "Zoom in and out the image (it doesn’t affect to the crop area but the image). Maximum zoom is 200% and minimum is 12,5%."

Q1: By "doesn’t affect to the crop area but the image," do you mean that it doesn't adjust the proportions of the cropping? If so, when I use the tool with a Free ratio, the visible area determines the cropped area. Is this the expected behavior.

Q2: Why is the minimum zoom 12.5%?

### CODE REVIEW

File: docs/dxp/latest/en/content-authoring-and-management/documents-and-media/uploading-and-managing/editing-images.md [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+
+When editing an image in Documents and Media, the following options are available.
+
+![Crop and rotate images.](./editing-images/images/02.png)
+
+**Crop**: Manually adust the cropping frame; click and drag the image to reposition it within the cropping frame.
+
+**Ratio**: Select a fixed preset ratio for cropping the image (i.e., 16:10, 16:9, 4:3, 1:1, 2:3), or select *Free* to create a custom image ratio.
+
+**Rotate**: Rotate image counter clockwise.
```
Comment (carloslancha) [inline docs/dxp/latest/en/content-authoring-and-management/documents-and-media/uploading-and-managing/editing-images.md:17]: Rotation of the image is clockwise

**Review by carloslancha — CHANGES_REQUESTED** [review]
Q: Will these features be added again in the future, or is there no plan to incorporate them again?

A: There won't be added, this is intended to be a minimal image editor with only crop and rotate functions.

Q: By "doesn’t affect to the crop area but the image," do you mean that it doesn't adjust the proportions of the cropping? If so, when I use the tool with a Free ratio, the visible area determines the cropped area. Is this the expected behavior.

A: What I mean is that the cropping area doesn't change it's size, ratio or anything when zoom changes, what is zoomed in and/or out is the image.

Q: Why is the minimum zoom 12.5%?

A: UX decission. cc @emiliano-cicero

File: docs/dxp/latest/en/content-authoring-and-management/documents-and-media/uploading-and-managing/editing-images.md (hunk 1 above)
Comment (JamesAGarcia) [reply docs/dxp/latest/en/content-authoring-and-management/documents-and-media/uploading-and-managing/editing-images.md:17]: Thank you or catching this!

**Comment by JamesAGarcia** [comment]
@carloslancha, I appreciate the input and feedback!

**Pull request closed without merging**